src/components/Card/README.md

```diff
@@ -436,6 +436,78 @@ Use to be able to use custom React elements as header content.
 </Card>
 ```
 
+### Full example of card
+
+<!-- example-for: web -->
+
+Use as a broad example that includes most props available to card.
+
+```jsx
+<Card
+  sectioned
+  title="Dashboard"
+  actions={[{content: 'View dashboard'}]}
+  secondaryFooterAction={{content: 'Dismiss'}}
+  primaryFooterAction={{content: 'Export Report'}}
+>
+  <Card.Header
+    actions={[
+      {
+        content: 'Total Sales',
+      },
+    ]}
+    title="Sales"
+  >
+    <Popover
+      active
+      activator={
+        <Button disclosure plain>
+          View Sales
+        </Button>
+      }
+      onClose={() => {}}
+    >
+      <ActionList items={[{content: 'Gross Sales'}, {content: 'Net Sales'}]} />
+    </Popover>
+  </Card.Header>
+  <Card.Section title="Total Sales Breakdown">
```

Comment:
DataTable inside a card section creates a lot of horizontal dividers that make the information hierarchy hard to understand. It would also be nice to show off some different types of content in the sections.

What about a `<Card.Section fullWidth>` with a simple `ResourceList`, and a couple other sections with some other content, like a paragraphs inside a `TextContainer`?